articles/documentation-website-hugo/index.md

```diff
@@ -0,0 +1,206 @@
+
+```bash
+$ hugo new site documentation-website
+
+Congratulations! Your new Hugo site is created in /Users/sheriff/hugo-project.
+
+Just a few more steps and you're ready to go:
+
+1. Download a theme into the same-named folder.
+   Choose a theme from https://themes.gohugo.io/ or
+   create your own with the "hugo new theme <THEMENAME>" command.
+2. Perhaps you want to add some content. You can add single files
+   with "hugo new <SECTIONNAME>/<FILENAME>.<FORMAT>".
+3. Start the built-in live server via "hugo server".
+
+Visit https://gohugo.io/ for quickstart guide and full documentation.
+```
+
+The `hugo new site` command will also generate a directory with the following file structure:
+
+```
+hugo-project/
+   ├── archetypes
+   ├── config.toml
+   ├── content
+   ├── data
+   ├── layouts
+   ├── static
+   └── themes
+```
+
+The following files are:
+- `archetypes`: Directory for storing content template files.
+- `config.toml`: Hugo configuration file.
+- `content`: Directory for content files.
+- `data`: Directory for storing configuration files.
+- `layouts`: Directory for storing .html template files.
+- `static`: Directory for storing all static contents.
+- `themes`: Directory for storing your theme.
+
+### Step 3 - Install the Docsy theme 
+
+The Docsy theme is a Hugo theme that is built specifically for creating medium to large technical documentation websites. The theme comes preloaded with basic documentation features like the search bar, the menu, etc.
+
+To install the Docsy theme:
+
+1. cd to your project's directory.
+```bash
+cd documentation-website
+```
+2. Install postCSS and autoprefixer .
+```bash
+sudo npm install -D --save autoprefixer
+sudo npm install -D --save postcss-cli
+```
+3. Install the Docsy theme by adding it into your as a git submodule.
+```bash
+git init
+git submodule add https://github.com/google/docsy.git themes/docsy
+echo 'theme = "docsy"' >> config.toml
+git submodule update --init --recursive
+```
+
+### Step 4 - Add a content section to your website
+
+All your website's content files are stored in the `content` folder found in your root folder. The top-level subfolders in the `content` folder will determine the sections of your website by default, and the nested subfolders inside these top-level subfolders combine to form your website structure as they are organized in their top-level subfolders.
+
+```
+content      <-- Content folder
+   |--- Documentation        <-- Top level subfolder(main section in your website).
+         |
+         |---- New Documentation    <-- Nested subfolder(subsection found in the documentation section).
+```
+         
+To add a custom section to your website, 
+
+1. cd to the `content` folder.
+```bash
+cd content
+```
+2. Create a new subfolder with the custom section name ie `documentation`.
+```bash
+mkdir documentation
+```
+3. Add the new section to your `menu` for easier navigation by editing your config.toml file.
+
+```config.toml
```

Comment:
please note that one windows it must be "start notepad++ .\config.toml" to edit since regular notepad or other common editing apps will encode the file incorrectly.